modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/RelatedObjectEntryResourceImpl.java

```diff
@@ -198,6 +226,24 @@ private ObjectDefinition _getRelatedObjectDefinition(
 			objectRelationship.getObjectDefinitionId2());
 	}
 
+	private ObjectEntry _getRelatedObjectEntry(
+			ObjectRelationship objectRelationship, long relatedObjectEntryId,
+			ObjectDefinition systemObjectDefinition)
+		throws Exception {
+
+		ObjectEntryManager objectEntryManager =
```

Comment:
Despite of I prefer to have little methods that do a specific task, I don't know if Brian would like this. Usually, if the method is just used once, he prefers to inline the method.
And, if my previous comment is right and you don't need to call this method inside the implementation of the delete endpoint, this method will be called once